src/VirtoCommerce.CatalogModule.Web/module.manifest

```diff
@@ -1,9 +1,9 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <module xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
   <id>VirtoCommerce.Catalog</id>
   <version>3.51.0</version>
   <version-tag />
-  <platformVersion>3.66.0</platformVersion>
+  <platformVersion>3.68.0</platformVersion>

```

Comment:
Could we avoid so unwanted dependency raising?  Duplicating a CSS style at the module level would be a good solution to avoid this.

Reply:
"unwanted"? Do we have problems regarding dependency raising? How much code duplication do you prefer? @Vectorfield4 @OlegoO , seems like an interesting topic.

Reply:
decided to merge without closing this comment. All other issues fixed

Reply:
I do not prefer any code duplication :)
To prevent any dependency raising from platform UI changes we need to talk about separating the platform styles from the platform. This is not very efficient to create "new" styles or duplicate UI content (icons, images, etc) every time we need to change UI in the modules.

@OlegoO FYI.